### DISCUSSION TITLE
Cubase setup steps with an extender

### DISCUSSION THREAD

**Discussion by danimusicos** [bjoluc/cubase-mcu-midiremote#1] [Q&A]
Bjr, j'ai suivi vos recommandation notamment en modifiant le script puisque j'utilise un extender placé à gauche de mon x touch principal. J'ai copié le fichier dans les répertoire décrits, j'ai supprimé les ancien dans cubase 12.0.52, mais après je ne comprends pas votre manipulation qui indique de faire + dans midi remote. Il n'y a pas + dans midi remote, il y a plus pour choisir les controleurs. Que voulez vous dire ? merci
daniel
trad google :
Bjr, I followed your recommendation in particular by modifying the script since I use an extender placed to the left of my main x touch. I copied the file in the directories described, I deleted the old ones in cubase 12.0.52, but afterwards I don't understand your manipulation which indicates to do + in midi remote.
There is no + in midi remote, there is more to choose the controllers. What do you mean ? THANKS
Daniel

**Comment by limageurpublic** [comment]
Hi,
is it ok if I have a 2 extenders + main X touch configuration ?
as : extender ( left ) + main ( center ) +extender ( right )
do I have to modify as : "devices: ["extender", "main", "extender"] ?
kind regards from France.

**Reply by danimusicos** [reply]
hi, i only use one extender. In any case except misunderstanding Cubase 12 does not detect anything Daniel

**Reply by bjoluc** [reply]
Hi @limageurpublic,

`devices: ["extender", "main", "extender"]` is right on! I silently added support for arbitrarily many extenders in v1.3.0 – silently because I only have one extender and cannot test the mapping with real devices. So it would be very helpful for me if you could test whether the second extender works properly and let me know if anything is odd – thanks in advance! :pray:

**Reply by limageurpublic** [reply]
Hi,
I'd love to help you testing with my 2 extenders configuration, but I'm a rookie and i fail to install your plugin inside cubase folder....clicking + in midiremote, and finding the script...the System doesn't find your script even correctly downloaded...would you please help me to succeed ? kind regards
Franck

**Reply by limageurpublic** [reply]
I've just checked my cubase version...12.0.20....do you think it could be the problem's source ?

**Reply by bjoluc** [reply]
> do you think it could be the problem's source

I hope so :sweat_smile: I didn't test the script with anything below 12.0.52.

**Reply by limageurpublic** [reply]
ok by clicking on "add a surface", i 've created a xtouch but "importing script" show a blank file...no script to load...would you please show a screen of your manip to add your proper script...

**Reply by bjoluc** [reply]
Just to validate that it's no problem with the file paths/script file – can you import this `.midiremote` file directly from within Cubase? [cubase-xtouch-midiremote-v1.3.0-extender-left.zip](https://github.com/bjoluc/cubase-xtouch-midiremote/files/10768519/cubase-xtouch-midiremote-v1.3.0-extender-left.zip)

**Reply by limageurpublic** [reply]
<img width="895" alt="Capture d’écran 2023-02-21 155536" src="https://user-images.githubusercontent.com/125523762/220379794-b6704a6f-ef15-49f2-920c-bdcd577d0e99.png">
<img width="731" alt="Capture d’écran 2023-02-21 155629" src="https://user-images.githubusercontent.com/125523762/220379798-0e8fc385-bb2c-4a88-9092-e44403e8fef0.png">

**Reply by limageurpublic** [reply]
I just saw your message...i'm really a rookie man...i try it and come back soon

**Reply by bjoluc** [reply]
Oh, thanks for the folder screenshot! By "[Download](https://github.com/bjoluc/cubase-xtouch-midiremote/releases/latest/download/behringer_xtouch.js) the MIDI Remote script file from the latest GitHub release" I meant only the `behringer_xtouch.js` file, not the full archive... So within ".../Behringer/xtouch" you should have `behringer_xtouch.js` then.

**Reply by limageurpublic** [reply]
<img width="620" alt="Capture d’écran 2023-02-21 160353" src="https://user-images.githubusercontent.com/125523762/220381547-056cbfd6-8a4b-423e-8c9b-36c07bfdfeda.png">
I did it but my xtouch and the left extender are usb connected and the second extender placed in right of the xtouch main, is in ethernet mode...so, by adding your script, cubase doesn't show an usb in/out for the xtouch, but a midiin/out ontly....it failed...

**Reply by bjoluc** [reply]
Everything looks good on your screenshot so far – only that the script I shared in the `.midiremote` file has `devices: ["extender", "main"]` only in the config, so it only shows two input/output MIDI ports. Can you change it to `"extender", "main", "extender"` in the script file and restart Cubase?

**Reply by limageurpublic** [reply]
i've put the two files, but when trying to import script...cubase shows only the "midiremote" files but not the### ".js" file...
<img width="548" alt="Capture d’écran 2023-02-21 161014" src="https://user-images.githubusercontent.com/125523762/220383393-6d5a307c-f321-4c86-bb19-20d1754bb063.png">

**Reply by bjoluc** [reply]
The .js file is already at the right place – no need to import it. All Cubase did when importing the .midiremote file was creating the .js file at that place... Now you should be able to edit the .js file to include the second extender, and restart Cubase.

**Reply by limageurpublic** [reply]
![1676992695112](https://user-images.githubusercontent.com/125523762/220385159-edea311f-20db-4b79-a72b-fd61bc04f7b2.jpg)
just a picture of my own config...with my little cript...but i'd love to use yours and the xtouchs could sho
w the tracks colors...

**Reply by limageurpublic** [reply]
how can i modify the js file ?

**Reply by limageurpublic** [reply]
ok, i just modyfied the script...and "tadaaam"...look at thy screenshot...but nothing "appers" on the 3 xtouchs...nothing reported...blank lcds...
![1676993440129](https://user-images.githubusercontent.com/125523762/220388127-51d359ee-be36-4246-93cf-6821982d9219.jpg)
<img width="1583" alt="Capture d’écran 2023-02-21 162838" src="https://user-images.githubusercontent.com/125523762/220388201-887a96c1-2d74-4933-886b-5e1c6926bbc7.png">

**Reply by limageurpublic** [reply]
<img width="193" alt="Capture d’écran 2023-02-21 163241" src="https://user-images.githubusercontent.com/125523762/220388711-e83943d9-9308-4565-b2e8-e3c6801c4c8b.png">
and it replies "no assignation" on any button on your script sketch

**Reply by limageurpublic** [reply]
Hope i could find a solution with your help in order to help others users...cause the xtouch is incredible and low cost solution for many music makers

**Reply by limageurpublic** [reply]
kind regards.

**Reply by bjoluc** [reply]
Cubase doesn't ask for MIDI ports again when the script is changed – you may need to disconnect the X-Touchs, remove the device surface, connect them, and add the surface again...

**Reply by bjoluc** [reply]
To anyone in a similar situation: The problem here was an unsupported Cubase version (12.0.20 ).

**Comment by limageurpublic** [comment]
Re,
Cubase accept the second extender which is ethernet connected...
Franck

Le jeu. 16 févr. 2023, 17:29, danimusicos ***@***.***> a
écrit :

**Reply by danimusicos** [reply]
moi je suis branché les deux en usb

**Reply by danimusicos** [reply]
https://github.com/bjoluc/cubase-xtouch-midiremote/discussions/1#discussioncomment-5184230
Salut, luc a modifié le script qui fonctionne sur cubase sauf les nuances de couleur qui ne donnent rien de plus que la version 1.3.0. Il m'a conseillé de voir avec vous. Merci Daniel

**Comment by bjoluc** [comment]
Hi @danimusicos,

thanks for reaching out here and helping to declutter the Steinberg forum thread a bit!

> Il n'y a pas + dans midi remote, il y a plus pour choisir les controleurs.

That plus to select a controller is exactly what I meant to refer to. The Readme description is not very clear about the manual setup steps – my apologies for that. Does [this manual entry](https://steinberg.help/cubase_pro/v12/en/cubase_nuendo/topics/midi_remote/midi_remote_midi_controller_surfaces_adding_t.html) help?

**Reply by bjoluc** [reply]
This is how it looks for me:

![manual-controller-setup](https://user-images.githubusercontent.com/25661029/219663348-8f6e6b58-87fc-4ed7-babe-370d3950f9f7.png)

**Reply by danimusicos** [reply]
Malheureusement, j'ai pourtant fait comme vous indiquiez dans la procédure, mis le fichier dans le nouveau dossier de steinberg. Mais rien de visible dans cubase. Et la fenêtre de configuraiton du xtouch, n'est pas possible.
![2023-02-17 15 50 56](https://user-images.githubusercontent.com/52532781/219687547-dc74a537-2d6b-43bf-ae30-e14cf1760981.png)

**Reply by bjoluc** [reply]
I think the manual refers to the lower zone's midi remote pane ("Ouvrir MIDI Remote dans la zone infé"). However, as your other screenshots reveal, the MIDI remote script is not recognized somehow, so using the lower zone alone won't help... If the `Local` folder and all sub folders are placed/named correctly, this may be due to a syntax error in the script that was accidentally introduced while editing the config. Does Cubase (after restarting it) list the script if you replace it with a fresh downloaded copy?

**Reply by danimusicos** [reply]
j'avais retéléchargé le fichier. Mais çà ne fait rien de particulier. Je vais tenter de re démarrer la machine. l'anti virus est totalement désactivé.

**Reply by bjoluc** [reply]
Ok, to simplify things a bit, here's one thing you can try: I exported v1.3.0 as a `.midiremote` file from within Cubase (with the extender configuration already done) and zipped it, so you should be able to unzip it and import the `.midiremote` file directly from within Cubase: [cubase-xtouch-midiremote-v1.3.0-extender-left.zip](https://github.com/bjoluc/cubase-xtouch-midiremote/files/10768519/cubase-xtouch-midiremote-v1.3.0-extender-left.zip)

**Reply by danimusicos** [reply]
çà fonctionne. Effectivement quelques omissions décrite sur votre description comme les boutons f1 à f8. le bouton display / name value semble ne plus fonctionner. (je n'ai pas vu de descriptif sur celui-ci) Vous savez à quoi servaient les boutons à côté d'edit qui vont de 1 à 8 ? 
je continue de tester, dites moi si vous avez des questions sur mon utilisation en tout cas les 2 machines fonctionnent parfaitement en apparence en tout cas

**Reply by danimusicos** [reply]
une critique peut-être pas de la création du script, mais la couleur rouge altère la lecture, ainsi que le bleu. le vert c'est impecable. Je vais tenter de régler les contrast des xtouch.  ensuite, les couleurs de cubases sont mal reproduites sur les controleurs, par exemple violet est rouge, en fait on dirait que seulement les couleurs primaires sont gérées

**Reply by bjoluc** [reply]
Haven't had any other reports about Key/Value not working yet. You're on Cubase 12.0.52, right?
The 1-8 buttons load visibility presets 1-8 like in the original Mackie Control mapping, but the Remote API doesn't yet reflect channel visibility presets. So as long as this isn't implemented in Cubase, feel free to assign the buttons to something else...
Re Colors: Since there are only 7 colors available on the X-Touch, the script maps each track color to the closest channel strip color in the RGB spectrum. Hence, there are always colors for which the closest X-Touch color is still rather inaccurate. Readability is indeed not optimal for all colors. It's ok for my use, but it strongly depends on the viewing angle.

**Reply by bjoluc** [reply]
> the script maps each track color to the closest channel strip color in the RGB spectrum. Hence, there are always colors for which the closest X-Touch color is still rather inaccurate. 

I was wrong about that: There was still a lot of room for improvement on my end, and I fixed color mapping in [Release 1.3.1](https://github.com/bjoluc/cubase-xtouch-midiremote/releases) – thanks for pointing this out!

**Reply by danimusicos** [reply]
hello, comme la première fois le scipt n'est pas reconnu dans cubase. Vous m'aviez adressé le fichier sou sforme de fichier midi remote

**Reply by bjoluc** [reply]
I sent you the midi remote file once to verify there were no problems with the script itself. Now that we know that the script works on your machine, I encourage you to try setting it up based on the setup instructions. Things you might check are:
* Is the script directory name precisely `xtouch`?
* Does the devices configuration in the script read `devices: ["extender", "main"],`?
If nothing helps, maybe you can contact @limageurpublic for help. He got the setup right and speaks French too – I can only offer English.

Best of success!

**Comment by danimusicos** [comment]
![2023-02-17 15 53 11](https://user-images.githubusercontent.com/52532781/219688175-078065f4-6311-4449-93c7-580046ffc065.png)

**Comment by danimusicos** [comment]
![2023-02-17 15 56 41](https://user-images.githubusercontent.com/52532781/219688901-f80d5c8e-ac6f-4fd2-9c33-26924abf9d5a.png)

**Reply by danimusicos** [reply]
le bouton scrub à un effet qui n'existait pas avant c'est à dire que dès qu'on s'en sert avec la molette, le bouton stop et avant de cubase gigottent en même temps.

**Reply by bjoluc** [reply]
Right you are. This is directly related to how the Cubase Remote API handles the play state, so sadly I can't change it...

**Comment by danimusicos** [comment]
Bonjour, Je remarque qu'à chaque fois que j'allume ma machine et que je relance cubase 12 et mon projet, la xtouch n'est jamais reconnue et je dois aller cher le script, y a til une manipulation à faire ?  
Hello, I notice that every time I turn on my machine and I restart cubase 12 and my project, the xtouch is never recognized and I have to go to the script, is there a manipulation to do?

**Reply by bjoluc** [reply]
Well, that's annoying. Unfortunately, I can't reproduce it :confused:

**Comment by danimusicos** [comment]
Salut, ok pour toutes les réponses. Je suis déstabilisé pour les fonctions que j'avais configurées sur les touche f1 à f8. Elle me manque et je n'arrête pas de les utiliser par habitude du coup çà ne donne plus l'effet voulu ! Mais je savais que c'était un problème déjà connu. A priori c'est déjà pas mal, vous êtes un champion.

**Reply by bjoluc** [reply]
> Je suis déstabilisé pour les fonctions que j'avais configurées sur les touche f1 à f8. Elle me manque et je n'arrête pas de les utiliser par habitude du coup çà ne donne plus l'effet voulu ! Mais je savais que c'était un problème déjà connu.

You mean the fact that the F1-F8 buttons cannot have a second assignment when the shift button is pressed? Without considering the shift button, you should be able to assign the F1-F8 buttons in the MIDI Remote Mapping Assistant to match the functions on your old Mackie Control setup. Or do you mean the channel visibility presets (Button 1-8) not being respected? I hope Steinberg implements this soon.

> A priori c'est déjà pas mal, vous êtes un champion.

Thanks :sweat_smile:

**Reply by danimusicos** [reply]
Salut, oui en fait, j'avais des fonction avec et sans shift pour des raccourcis utiles. Par contre, je n'ai pas compris comment fallait procéder ?

**Reply by bjoluc** [reply]
You can assign functions to buttons with the [MIDI Remote Mapping Assistant](https://steinberg.help/cubase_pro/v12/fr/cubase_nuendo/topics/midi_remote/midi_remote_mapping_pages_r.html). Note that the script is inactive while the assistant is open, i.e., you need to close the assistant for your mapping to take effect.

**Comment by danimusicos** [comment]
salut, je l'ai compris avant de lire votre message c'est nickel, çà fonctionne.

**Comment by limageurpublic** [comment]
Hello,
Sous cubase, ça reste instable.. Hier ça marchait, puis ça marchait plus
après avoir fermé puis ré ouvert cubase, et ce matin, avec un vieux projet
cela ré fonctionnait parfaitement.
Comment puis-je t'aider ?... Sachant que Bjorn m'a dit de ne pas diffuser
le script sans son autorisation, je ne veux pas lui faire "un petit dans le
dos"... Mais je serais ravi de t'aider...
À suivre...

**Comment by limageurpublic** [comment]
Hi Bjorn,
I've downloaded the last version 1.4 ( i changed "extender, main, extender" to be like my devices setup ) but the scribbles of the first left extender  and main xtouch don't show the color of the cubase tracks....the second extender on the left shows correctly the tracks colors...amazing...
Hope you're fine.
Kind regards.

**Reply by limageurpublic** [reply]
I'll be available by the end of the day or tomorrow if you want to test it

**Reply by danimusicos** [reply]
Salut, je vais tester de mon côté.

**Reply by limageurpublic** [reply]
![Uploading 2023-03-15 15.30.15.jpg…]()

**Reply by danimusicos** [reply]
> I'll be available by the end of the day or tomorrow if you want to test it

no problem for my

**Comment by limageurpublic** [comment]
11.00am today?

**Reply by danimusicos** [reply]
Aucun problème avec la version 1.4, tout est OK. Pas vu d'anomalie. J'ai x touch et extender.

**Comment by limageurpublic** [comment]
![IMG_20230316_093207](https://user-images.githubusercontent.com/125523762/225559622-9156eea4-f6d3-4e63-85de-fcefa4ee665c.jpg)
During next meeting, is it possible to "copy" my own shortcuts as shown into your script manually ( and explain me how to do it alone without boring you.. Lol...) because my issues are, perhaps, caused by the two scripts running together... See you

**Reply by danimusicos** [reply]
Hello, quand on cliques sur F1 par exemple, on ouvre la configuration d'interface de cubase permettant d'assigner les raccourcis normalement ! Dès que je serai sur mon pc, je te ferai une copie d'écran.

**Reply by limageurpublic** [reply]
![Capture d’écran 2023-03-16 113213](https://user-images.githubusercontent.com/125523762/225590917-0e324cbe-3d1c-4af1-b475-29da82c1af85.jpg)
Hello,
1. j'ai supprimé le script de ma Xtouch, j'ai seulement laissé le script de Bjorn, et les LCD etc fonctionnent.
2. en revanche, j'ai parametré les deux rangées de boutons et , bien que cubase a bien enregistré les raccourcis dans la surface de contrôle, rien de se passe quand j'appuie sur l'un des 16 boutons...mystère...
à suivre...

**Comment by limageurpublic** [comment]
![1678962509599](https://user-images.githubusercontent.com/125523762/225591494-56e6dbe4-4a1c-41a2-a4c6-f389d4d9a77f.jpg)
![1678962509616](https://user-images.githubusercontent.com/125523762/225591794-db294cd6-6582-49d5-b54b-661f0f78b2fd.jpg)

lorsqu’on choisit ces deux nuances de bleu/violet, le nom de la piste est illisible ( comme si on avait diminué la luminosité )

**Comment by limageurpublic** [comment]
Hi,
New issue...it seems like the script or cubase api "lost" the midi ports defined at the set up !
![Capture d’écran 2023-03-16 143443](https://user-images.githubusercontent.com/125523762/225634052-77a37642-3cba-4e96-ba0c-7756de73f5ea.jpg)
![Capture d’écran 2023-03-16 143517](https://user-images.githubusercontent.com/125523762/225634078-e474fe5d-810e-49d5-a2e2-81d87b35a6e1.jpg)
the midi ports were in and out in the same order at set up ... xtouch ext, xtouch, and Desktop....
but at the next start up of cubase project...the midi out ports seem to be unordered and, at this moment, the first extender ( called stouch-ext ) doesn't work properly anymore....it show the tarcks names but not colors....

**Reply by limageurpublic** [reply]
Hope i'm not boring you, but only help you to improve your script
kind regards

**Comment by bjoluc** [comment]
Hi guys,

I'm super happy that you help each other out! However, having been away for a few days, I've totally lost track of all the threads that are going on here. If you are facing issues with the script, feel free to open [issues](https://github.com/bjoluc/cubase-xtouch-midiremote/issues) which are separate threads used to track problems and their solutions. For questions, you can also open other discussions since this topic was originally intended to be about @danimusicos' setup issues with a single extender... If you'd like to explicitly address me (so I know you mean *me*), you can also tag me (@bjoluc).

Thanks!

**Reply by danimusicos** [reply]
Salut, OK pour votre recommandation.
À bientôt.

**Reply by limageurpublic** [reply]
Hello Dani, 
je ne sais pas comment t'envoyer un message directement...quelle genre de musique fais tu ?
Bonne journée.
Franck

**Reply by danimusicos** [reply]
Salut, je fais de la musique pour le plaisir dite de variété ou populaire.
@Danielaime sur youtube

**Comment by limageurpublic** [comment]
Hi Bjorn,
Did I made a mistake or "when changing tracks visibility" in "console", nothing change on devices, all channels/tracks still be present ?
Would you please look at it.
kind regards.
Hope you're fine.

PS : as usual, due to cubase lack, when crating a new project, the shortcuts on xtouch are disables...i have to recreate a surface in midiremote tab and everything goes now properly...grrr...lol

**Reply by bjoluc** [reply]
> Did I made a mistake or "when changing tracks visibility" in "console", nothing change on devices, all channels/tracks still be present ?

Hi Franck,
you did everything right – it's a [known limitation](https://github.com/bjoluc/cubase-xtouch-midiremote#drawbacks) at the moment and I'm waiting for a Cubase release that enables this...

**Comment by limageurpublic** [comment]
Re,
In fact, I am very happy to use your script and make my devices quite fully
functional.
It's a pleasure to ease my work flow due to your work.kind regards.

**Comment by limageurpublic** [comment]
Hi Bjorn,

How are you ?
I installed my homestudio in a new town, and I am coming back , lol.

I've just downloaded the 1.8.0 js and put it in my new CUBASE 13 version...
Unfortunately, the lcd on the xtouch doesn't reply the cubase channel colors, it only put "default grey color" for any channel as you can see in attached file, but the photo is bad quality.

Did I make a mistake during install ?
Kind regards.
![1701287719022](https://github.com/bjoluc/cubase-mcu-midiremote/assets/125523762/2f23e0ac-9f94-4699-84dd-ed7a7b569d60)

**Reply by bjoluc** [reply]
Hi Franck!

Sorry for the late reply, I've had a full past month and am only getting back to this now. Did you manage to solve your color scribble strip issue? No one else reported problems with this, so I assume something is wrong with your setup...

**Comment by danimusicos** [comment]
Salut,

Ca me le fait de temps à autre. Il faut recommencer l'installation dans cubase. Donc désactiver le script le réactiver et remettre xtouch

Tiens je t'adresse mon souci, il sera intéressant de voir si çà le fait également chez toi.

je t'adresse la vidéo du probléme par


________________________________
De : limageurpublic ***@***.***>
Envoyé : vendredi 1 décembre 2023 12:02
À : bjoluc/cubase-mcu-midiremote ***@***.***>
Cc : danimusicos ***@***.***>; Mention ***@***.***>
Objet : Re: [bjoluc/cubase-mcu-midiremote] Cubase setup steps with an extender (Discussion #1)


Hi Bjorn,

How are you ?
I installed my homestudio in a new town, and I am coming back , lol.

I've just downloaded the 1.8.0 js and put it in my new CUBASE 13 version...
Unfortunately, the lcd on the xtouch doesn't reply the cubase channel colors, it only put "default grey color" for any channel as you can see in attached file, but the photo is bad quality.

Did I make a mistake during install ?
Kind regards.
1701287719022.jpg (view on web)<https://github.com/bjoluc/cubase-mcu-midiremote/assets/125523762/2f23e0ac-9f94-4699-84dd-ed7a7b569d60>

**Reply by limageurpublic** [reply]
Hello,
Bonnes fêtes de fins d'année...
Je crois que j'ai zappé ton message...sorry
Je voudrais bien réinstaller mais je pose une question supp dans mon post " les shortcuts ajoutés perso disparaissent avec l'install du nouveau script....donc, je voudrais bien savoir où "ou" comment sont stockés les "shortcuts" pour ne pas me retaper toute la config des 16 boutons (F1 à F8x2)"...je suis peut-être pas très doué, j'en conviens.
à suivre...

**Comment by danimusicos** [comment]
salut, le bouton beat ou smpt ne fonctionne pas bien avec cubase 12.070, je joins la vidéo qui montre l'anomalie, çà le faisait sur la 1.70
même en changeant sur cubase c'est pareil la xtouch ne change pas.

https://github.com/bjoluc/cubase-mcu-midiremote/assets/52532781/f5f7a076-71f1-42a8-b5cb-a599557635c2

**Reply by limageurpublic** [reply]
j'avoue ne pas utiliser cette fonction...je suis un peu fainéant pour essayer d'apprivoiser totalement la Xtouch...et puis je ne suis pas un pro de cubase non plus...rires...

**Comment by danimusicos** [comment]
Salut, idem, même problème ! Par contre, avec cubase 13, problème de perte
de fonctions. Par exemple, quand on choisi une piste, le témoin de piste
n'apparaît pas du coup on ne sait pas sur quelle piste on se trouve.



Le mer. 27 déc. 2023 à 07:50, limageurpublic ***@***.***> a
écrit :

**Comment by danimusicos** [comment]
Merci pour ta réponse !

Le mer. 27 déc. 2023 à 07:52, limageurpublic ***@***.***> a
écrit :

**Comment by danimusicos** [comment]
Salut, pour moi pas de soucis avec cela.

Par contre, la façon de programmer les fonctions a changé et c'est moins pratique surtout quand on utilise une pédale de contrôle sur le xtouch.

Les touches de la x touch ne s'allument plus toujours il faut s'y reprendre à plusieurs fois et appuyer plusieurs fois.

J'ai mis çà sur le suivi

Daniel

Hi, for me no problem with this.

On the other hand, the way of programming the functions has changed and it is less practical, especially when using a control pedal on the xtouch.

The x touch keys no longer always light up, you have to try them several times and press them several times.

I put this on the follow-up

Daniel
________________________________
De : bjoluc ***@***.***>
Envoyé : vendredi 12 janvier 2024 16:22
À : bjoluc/cubase-mcu-midiremote ***@***.***>
Cc : danimusicos ***@***.***>; Mention ***@***.***>
Objet : Re: [bjoluc/cubase-mcu-midiremote] Cubase setup steps with an extender (Discussion #1)


Hi Franck!

Sorry for the late reply, I've had a full past month and am only getting back to this now. Did you manage to solve your color scribble strip issue? No one else reported problems with this, so I assume something is wrong with your setup...

**Comment by danimusicos** [comment]
Sujet clos
